e2e/e2e.go

```diff
@@ -192,6 +197,78 @@ func main() {
 
 	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
 
+	// runEndToEndTests is a flag that denotes whether or not this needs to actually
+	// run or not based off of the filepath.Walk function immediately proceeding.
+	var runEndToEndTests bool
+
+	err := filepath.Walk(".", func(path string, info os.FileInfo, err error) error {
```

Comment:
I think it might be simpler to use [build.ImportDir](https://pkg.go.dev/go/build#Context.ImportDir) and inspect the [Package.AllTags](https://pkg.go.dev/go/build#Package).   If you decide to just keep the current structure, I suggest atleast using [constraint](https://pkg.go.dev/go/build/constraint@go1.16.6) to check if a line is a build constraint.

Reply:
Oh I didn't see this before hitting the merge button. I'll change the code to use the `build.ImportDir` function whenever I get some free time next, that seems like a better way to do this, I agree. I'll create a ticket to track this

Reply:
https://outreach-io.atlassian.net/browse/DTSS-605